dashboard/app/models/lti_user_identity.rb

```diff
@@ -9,10 +9,12 @@
 #  created_at         :datetime         not null
 #  updated_at         :datetime         not null
 #  deleted_at         :datetime
+#  issuer             :string(255)      not null
```

Comment:
This is already stored in the LtiIntegration associated with the LtiUserIdentity. Can we use that instead?

Reply:
There is no direct association between `User` and `LtiIntegration`.
Given an `id_token` received by an LTI exchange, we would have to use the `authentication_options` model.

The `lti_user_identity` is the only model that fully maps `Users` to `LTI Integrations`

This model was built early in the project and we did not consider the possibility of collisions.
It is also logical to include the issuer of an Identity in the model.